frontend/src/scenes/insights/utils.tsx

```diff
@@ -617,3 +620,14 @@ export function sortDates(dates: Array<string | null>): Array<string | null> {
 export function getResponseBytes(apiResponse: Response): number {
     return parseInt(apiResponse.headers.get('Content-Length') ?? '0')
 }
+
+export const insightTypeURL: Record<InsightType, string> = {
```

Comment:
So `urls.insightNew()` shouldn't be used directly? 🧐 That seems like a redundant level of complexity. Shouldn't we do this branching _inside_ `insightNew` so that there's only one way of doing this?

Reply:
There are 48 calls to insightNew either because they're passing a specific `Filter` or are passing nothing happy to get the default.

This was only to pull into one place "please can I have a new insight of a specific type with defaults" as opposed to "can I have the default new insight" or "can I have this specific new insight"

Many of the calls will change  as we swap from `Filter` and `Query`